android/build.gradle

```diff
@@ -26,9 +26,9 @@ android {
 }
 
 dependencies {
-  api project(':jdk')
+  api project(':common')
```

Comment:
Keep the `jdk`.

I just had an android library with just the `common` artifact and couldn't call anything until I added `jdk` as well. `jdk` artifact does some magic so the whole thing can be consumed by other java modules.